src/main/java/clevernotbot/AddEventCommand.java

```diff
@@ -0,0 +1,54 @@
+package clevernotbot;
+
+/**
+ * Represents a command used to add an Event Task.
+ */
+public class AddEventCommand extends Command {
+
+    /**
+     * Constructor for the AddEventCommand.
+     *
+     * @param commandName Description of Command.
+     * @param exit        Checking if program intends to exit.
```

Comment:
Good use of alignment to keep description on one side and parameters on the other